collectors/ebpf.plugin/README.md

```diff
@@ -219,10 +219,40 @@ The following options are available:
     range of IPs, or use CIDR values. The default behavior is to only collect data for private IP addresess, but this
     can be changed with the `ips` setting.
     
-By default, Netdata displays up to 500 dimensions on network viewer charts. If there are more possible dimensions, they
+By default, Netdata displays up to 50 dimensions on network viewer charts. If there are more possible dimensions, they
```

Comment:
It needs to be clear what the criterion that qualifies which dimension will be displayed is. Is it the most recent one that displaces the rest? Is it the most frequently used ones? I guess in our case it would be the oldest dimensions are displayed and the newest are bundled into `other`?

Reply:
I changed the paragraph to explicit that we are showing on `1.24` the oldest one.